ckanext/scheming/plugins.py

```diff
@@ -83,13 +83,21 @@ def _add_template_directory(self, config):
     def _load_presets(self, config):
         if _SchemingMixin._presets is not None:
             return
-        presets = config.get('scheming.presets', DEFAULT_PRESETS).split()
+        presets = config.get('ckanext.scheming.presets', DEFAULT_PRESETS).split()
         _SchemingMixin._presets = {}
         for f in reversed(presets):
             for p in _load_schema(f)['presets']:
                 _SchemingMixin._presets[p['preset_name']] = p['values']
 
     def update_config(self, config):
+
+        # Check for old 'scheming.*' config options
+        new_options = {}
+        for key, value in config.iteritems():
+            if key.startswith('scheming.'):
+                new_options['ckanext.' + key] = value
+        config.update(new_options)
```

Comment:
@wardi ok ok, I can add another check to see if there's an existing `'ckanext.' + key` before  overwriting it :)

The whole PR is no big deal, I'm happy to drop it if you want, I just like extensions config options to start with `ckanext.`